source/docs/use/my.md

```diff
@@ -1,23 +1,21 @@
 ---
-id: my
 title: 'My Home Assistant'
-description: 'My link support'
+description: 'Creating a My link to a HACS repository'
 ---
 
-!!! note
+**Home Assistant My links** can be used to link directly to a repository inside HACS. This is used for example, to share a link on Discord or to bookmark it. My links work for both default and custom repositories.
 
-      This require at least these versions:
+## To create a My link using a hotkey
 
-      - Home Assistant 2022.8.0
-      - HACS 1.26.0
+1. In Home Assistant, open the repository of interest and press the `m` key.
+      - The My link page opens, with the information already filled in. 
+2. Copy the URL or snippets you want for your My link.
 
-      And you need to have the [`My` integration](https://www.home-assistant.io/integrations/my/)
+## To create a My link manually
 
-
-You can use [My](https://my.home-assistant.io) to link directly to an integration inside HACS.
-This works for both default and custom repositories.
-
-1. Start by going to https://my.home-assistant.io/create-link/?redirect=hacs_repository
-1. Fill in the fields on that page to generate your My link.
-   1. "Category" is only needed for custom repositories
-1. Copy the URL or snippets you want for your My link.
+1. Go to https://my.home-assistant.io/create-link/?redirect=hacs_repository.
+2. Fill in the fields on that page to generate your My link.
+   - **Owner**: the author of the repository. In the example of spook, this is "frenck"
+   - **Repository**: the name of the repositroy. For example: "spook"
+   - **Category** is only needed for custom repositories. For example: "integration".
```

Comment:
I have no good suggestion here, the field is named "category" on that site, so that needs to stay.
But this "category" is what we refer to as "type" in HACS going forward.